Profiler/CProfilerCallback.cpp

```diff
@@ -338,8 +278,6 @@ HRESULT CProfilerCallback::AssemblyLoadFinishedImplementation(AssemblyID assembl
 		return S_OK;
 	}
 
-	EnterCriticalSection(&callbackSynchronization);
```

Comment:
We definitely have to synchronize assembly load. These can be loaded from different threads and we were already getting "murks" in the trace due to this ;)